presto-product-tests/src/main/java/io/prestosql/tests/jdbc/TestLdapPrestoJdbc.java

```diff
@@ -81,7 +82,7 @@ public void shouldFailQueryForWrongLdapPassword()
     @Test(groups = {LDAP, PRESTO_JDBC, PROFILE_SPECIFIC_TESTS}, timeOut = TIMEOUT)
     public void shouldFailQueryForWrongLdapUser()
     {
-        expectQueryToFail("invalid_user", ldapUserPassword, "Authentication failed: Access Denied: Invalid credentials");
+        assertTrue(expectQueryToFailAndReturnExceptionMessage("invalid_user", ldapUserPassword).startsWith("Authentication failed"));
```

Comment:
Do we need `expectQueryToFailAndReturnExceptionMessage` at all?
I would rather have the test verify full failure message, just as it used to do.